index.html

```diff
@@ -2,230 +2,441 @@
+    <header class="landing-page">
+      
+      
+      <div class="mobile-container">
+        <div>
+          <h2 class="logo">My Logo</h2>
+        </div>
 
-      <ul class="media-icon">
-        <li>
-          <a href="#"><img src="images/twitter.svg" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images/Linkedin icon.png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images/Vector (1).png" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images/github.svg" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="images/Vector (3).png" /></a>
-        </li>
-      </ul>
-    </section>
-    <section class="Works">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        
+        <nav class="menu-mobile">
+          <ul class="nav-links-mobile">
+            <li>
+              <a href="#portfolio" class="a-links">Portfolio</a>
+            </li>
+            <li>
+              <a href="#about__me" class="a-links">About</a>
+            </li>
+            <li>
+              <a href="#contact"class="a-links">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        
+      </div>
+
+     
+      <div class="desktop-container">
+          
+        <div>
+          <h2 class="logo">My Logo</h2>
+        </div>
+        
+        <nav class="desktop-nav-menu">
+          <ul class="nav-tabs">
+            <li class="tab-links"><a href="#">Portfolio</a></li>
+            <li class="tab-links"><a href="#">About</a></li>
+            <li class="tab-links"><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        
+      </div>
+
+
+      <section class="headline">
+          <h1 class="headline-text" id="headline-text">I’m Shalini Nayan <br/>Glad to see <br class="break" />you!</h1>
+          <p class="headline-paragraph" id="headline-paragraph">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p class="heading-social" id="heading-social">LET’S CONNECT</p>
+          <ul class="icons" id="icons">
+            <li id="twitter">
+              <img src="images/twitter.png" alt="twitter"/>
+            </li>
+            <li>
+              <img src="images/Linkedin.png" alt="linkedin"
+              />
+            </li>
+            <li>
+              <img src="images/github.png" alt="github"
+              />
+            </li>
+            <li>
+              
+            </li>
+            <li>
+              <img src="images/angellist.png" alt="angellist"
+              />
+            </li>
+          </ul>
+        </header>
+      </section>
```

Comment:
I love your design 👍🏽 but it lacks alignment, the elements would look better if they were vertically aligned, this is what you have so far 👇🏽 

![Screenshot 2023-03-31 at 01 03 49](https://user-images.githubusercontent.com/45739963/229027478-a18f5612-6bf1-4e2b-96ed-56d97c15799e.png)

Reply:
Noted and thank you for your kind suggestion.